apps/ospTutorial/ospTutorial.cpp

```diff
@@ -560,9 +611,10 @@ void _loadPBF(Server &server)
           == esriPBuffer::
               FeatureCollectionPBuffer_GeometryType_esriGeometryTypePoint) {
         const auto &coords = feature.geometry().coords();
-        const auto origin = server.updateOrigin(coords[0], coords[1], 0);
-        const auto position = vec3d(coords[0], coords[1], 1);
-        centers.emplace_back(position - origin);
+        auto position = vec3d(coords[0], coords[1], 1);
```

Comment:
geometry might have z? otherwise, why not 0?

Reply:
yes, I can fix the z issue. I think they even have an indicator whether or not the feature has a z value. will fix

For some reason the spheres don't show up if I put z=0. I don't really understand this part. Maybe they are just not visible from whatever angle we look at them. I also noticed that the frame looks different depending on what I put as z value.

Reply:
I don't like it either, see below. Let's discuss it in VC in a minute